solves for makerpm of pear 1.4.1

error: line 25: Dependency tokens must begin with alpha-numeric, '_' or '/': Conflicts: php-pear-PEAR_Frontend_Web != 0.5.0

but is not a real fix i guess.
not sure does $dep['conflicts'] perhaps should have different value than empty string.

--- ./PEAR/Command/Package.php~	2005-09-29 00:07:35.000000000 +0300
+++ ./PEAR/Command/Package.php	2005-09-29 00:07:37.000000000 +0300
@@ -1010,8 +1009,8 @@
                                     $ex = array($ex);
                                 }
                                 foreach ($ex as $ver) {
-                                    if (isset($dep['conflicts'])) {
-                                        $conflicts[] = $package . ' != ' . $ver;
+                                    if (isset($dep['conflicts']) and $dep['conflicts']) {
+                                        $conflicts[] = $package . ' != ' . $ver; # XXX: not valid .spec syntax
                                     } else {
                                         $conflicts[] = $package . ' = ' . $ver;
                                     }
